deploy/compose/.env.example

```
@@ -1,10 +1,10 @@
-POSTGRES_USER=dhadgar
-POSTGRES_PASSWORD=dhadgar
-POSTGRES_DB=dhadgar_platform
-
-RABBITMQ_DEFAULT_USER=dhadgar
-RABBITMQ_DEFAULT_PASS=dhadgar
-
+POSTGRES_USER=dhadgar
+POSTGRES_PASSWORD=dhadgar
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)